components/widgets/menu/menu.jsx

```diff
@@ -13,6 +13,7 @@ export default class Menu extends React.PureComponent {
         openUp: PropTypes.bool,
         id: PropTypes.string,
         ariaLabel: PropTypes.string.isRequired,
+        customStyles: PropTypes.object,
```

Comment:
Minor nit, but it might be better to just call this `style` since its behaviour seems to match the `style` prop used on plain HTML elements

Reply:
I somewhat agree, but because the component already had some styling and it's a global component I wanted to override it completely in this one instance without affecting the other instances.

Reply:
I'm not sure I understand. If we're just passing the inline style to this component as a prop, it wouldn't affect other instances.

Reply:
Sorry, I'll try to clarify. I meant I didn't want to merge these conditional styles with the ones I'm passing in for this particular case:
```
if (openLeft && !isMobile()) {
            styles.left = 'inherit';
            styles.right = 0;
        }
        if (openUp && !isMobile()) {
            styles.bottom = '100%';
            styles.top = 'auto';
        }
```
 In my case, I wan't this applied all the time regardless of the `!isMobile()` condition:
```
const customStyles = {
                left: 'inherit',
                right: 0,
                bottom: '100%',
                top: 'auto',
};
```

Reply:
Oh, okay. That makes sense then